Src/java/elm/src/main/java/org/cqframework/cql/elm/serializing/ElmLibraryWriter.java

```diff
@@ -0,0 +1,14 @@
+package org.cqframework.cql.elm.serializing;
+
+import org.hl7.elm.r1.Library;
+
+import java.io.*;
+import java.net.URI;
+import java.net.URL;
+
+public interface ElmLibraryWriter {
+
+    public void writeValue(Library library, Writer writer) throws IOException;
+
+    public String writeValueAsString(Library library);
```

Comment:
If the Reader interface is just `read` methods, I would use just `write` methods here for consistency.

Reply:
Resolved on [26af72a](https://github.com/cqframework/clinical_quality_language/pull/760/commits/26af72a105ec8e5ec0e4c4eb25f0ea0b2f7d9eb4)